sagan-site/src/main/resources/templates/layout.html

```diff
@@ -13,7 +13,12 @@
   <script th:inline="javascript">
         var baseUrl = /*[[@{/}]]*/ '../static/';
   </script>
-  <script data:curl-run="@{/run.js}" th:src="@{/lib/curl/src/curl.js}" src="../static/lib/curl/src/curl.js" async ></script>
+
+  <script src="/jspm_packages/system.js"></script>
```

Comment:
@bclozel Are you wanting to keep this stuff hooked up with data:curl-run? Full disclosure, I left it out here because A. it's simpler for this rough commit and B. I need to spend a little bit looking at + familiarizing myself with that.